examples/scale/backend/backend.go

```diff
@@ -40,19 +40,27 @@ var (
 
 	activeScenario = scenarios.ActiveScenario
 	statProcessor  = scenarios.NewStatProcessor()
+
+	mIterations          = telemetry.Counter("scale_backend_iterations", "fetch match iterations")
```

Comment:
we can actually use prometheus query to calculate these rates without adding these metrics. But for the sake of fast iterations I'll approve this PR. Can you add a TODO here so that we could clean it up later?

e.g. If you want to query FetchMatches error rate
`rate(grpc_io_server_completed_rpcs{grpc_server_method="openmatch.Backend/FetchMatches", grpc_server_status!~"OK"}[5m])`

Reply:
I specifically wanted to only use metrics from the point of view from the caller.  This way, eg, errors for calls which never even make it to the service are accounted for.

I just seemed easier for me conceptually to add these.